src/main/java/hello/Main.java

```diff
@@ -0,0 +1,36 @@
+package hello;
+
+import java.util.Optional;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import javax.script.Compilable;
+import javax.script.ScriptEngineManager;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+
+/**
+ * Created by Vladyslav Usenko on 16.01.2016.
+ */
+@SpringBootApplication
+public class Main {
+
+	ScriptEngineManager manager = new ScriptEngineManager();
+
+	@Bean Compilable scriptEngine() {
+		return Optional.ofNullable((Compilable)manager.getEngineByName("nashorn")).orElseThrow(()-> new IllegalStateException("Cannot initialize nashorn"));
```

Comment:
we cannot be sure that nashorn engine exists, we need clear exception message in this case